script.js

```diff
@@ -1,4 +1,79 @@
+/*
+---REQUIREMENTS---
+[x] Program should be able to take in a text input "scissors", "paper", "stone"
+[x] Program randomly chooses scissors, paper, stone
+[x] User input and program choice can be compared and determined whether user wins, loses or draw
+[x] After comparing user and program choices, frontend page will display the result
+[ ] Program should be able to validate that the input falls within the 3 options
+[ ] Format the output to make it look nicer.
+[ ] 
+*/
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = '';
```

Comment:
the first (var myOutputValue = 'hello world')  is not used so we dont have to keep it inside.
you can declare myOutputValue with var myOutputValue ;  without assigning a value first in this case. as the type of the var will change when you assign values to it